JSDemos/Demos/DataGrid/GroupSummaries/description.md

```diff
@@ -1 +1,17 @@
+
+### Position a Group Summary
+
+Group summaries are displayed in parentheses after the group caption. You can reposition them as follows:
+
+* Display a summary in a column          
+You can align a summary with the column by which it was calculated (see the Sale Amount and Total Amount columns). To do this, enable the [alignByColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#alignByColumn) property. If you want to align a summary with a column of your choice, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+* Move a summary to the group footer       
+To position a summary in the group footer, enable the [showInGroupFooter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInGroupFooter) property. The summary will be aligned with the column by which it was calculated (see the Total Amount column). If you want to use another column, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+### Format Summary Values
+You can format a summary value in the [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#valueFormat) property. This demo specifies the *"currency"* **valueFormat** for the Total Amount and Sale Amount summary values. You can also use the [displayFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#displayFormat) property to set a custom display format for summary values. For example, you can add custom text to a summary value, as shown in this demo for summaries of the *"sum"* and *"count"* types.
+
+### Sort Groups by Summary Values
+The DataGrid sorts groups by their captions. To sort them by summary values, populate the [sortByGroupSummaryInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/sortByGroupSummaryInfo/) array with objects. Each object should include the [summaryItem](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/sortByGroupSummaryInfo/#summaryItem) property that specifies a group summary item by which to sort groups. In this demo, the **sortByGroupSummaryInfo** array contains a single object that sorts groups by the number of orders.
```

Comment:
`The DataGrid sorts groups by their captions` - a "caption" is the wrong word to use here. Also, this first sentence describes the "default" state of things. 

We could say something like this: 

"The grid always sorts data groups. The default sort order depends on the corresponding group column values. For example, if you group data by Country, then groups will be sorted alphabetically by Country names. 

You may also want to sort group rows by their corresponding summary values. For example, you can sort countries by the number of fulfilled orders. To do that, use the ... "

But in the end, that's probably too basic. Then why don't we compress the text to the following fragment:

```suggestion
This demo sorts groups by summary values (the number of orders). To do this, the code adds the corresponding summary item to the [sortByGroupSummaryInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/sortByGroupSummaryInfo/) array. 
```

Reply:
The suggested text doesn't mention that the sortByGroupSummaryInfo array must consist of objects and that each object must specify the summaryItem property.
Also, the code doesn't "add" anything - it's the user who does that. But let's see what the final edit is.
Can the paragraph be
"This demo sorts groups by summary values (the number of orders). To do this, add an object to the [sortByGroupSummaryInfo]() array and specify the summary of type "count" in the summaryItem property."?

Reply:
Maybe I misunderstood something, but I disagree about "add". User (I assume you mean developer) writes code. Code (a line in that code, a command) adds an item to the array. A user cannot add an item to an array. I can say that figuratively, but it doesn't mean that "the code does something" is incorrect. If you have a code sample, you can always say "the following code does X". 

In your suggestion, you say "To do this, add". As I understand, the demo already does this. We need to instruct the customer to look into code, not to write code. 

Here's another suggestion then...

```suggestion
This demo sorts groups by summary values (the number of orders). To do this, the code adds an object to the [sortByGroupSummaryInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/sortByGroupSummaryInfo/) array. The object specifies the [summaryItem](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/sortByGroupSummaryInfo/#summaryItem) whose values are used to sort the group rows. 
```